classes/controller/ModuleAdminController.php

```diff
@@ -112,7 +112,9 @@ protected function l($string, $class = null, $addslashes = false, $htmlentities
                 $class = get_class($this);
             }
 
-            $translated = Translate::getModuleTranslation($this->module->name, $string, $class, null, $addslashes);
+            $fallback = false; // new translation system has already been tested
+            $escape = false; // don't convert special chars to html entities
```

Comment:
Maybe the `$escape` should be using the `$htmlentities` value?

Reply:
:thinking: I am not sure to understand your comment, do you mean that I should change variable name ?

Reply:
No, I mean that maybe `$escape` should be dependent of the value of `$htmlentities`.
```php
$translated = Translate::getModuleTranslation($this->module->name, $string, $class, null, $addslashes, null, $fallback, $htmlentities);
```

Reply:
So scatterbrained xD
I even didn't notice that argument ... I'll check in our specific case but it's default value at true looks a problem ^^'

Thanks for the double check ^^'

Reply:
I got the problem.

In fact, `$htmlentities` option is applied twice.
I'll review the workflow. Maybe it is better to remove it from `ModuleAdminControllerCore::l(...)` function.